dockers/docker-fpm-frr/start.sh

```diff
@@ -31,6 +31,11 @@ rm -f /var/run/rsyslogd.pid
 
 supervisorctl start rsyslogd
 
+# start eoiu pulling, only if configured so
+if [ $(sonic-cfggen -d --print-data | grep 'bgp_eoiu' | grep -c 'true') -ge 1 ]; then
```

Comment:
`sonic-cfggen -d -v bgp_eoiu` could work better here

Reply:
Hi Pavel, thanks for your review. However,  "sonic-cfggen -d -v" doesn't seem to do the desired check:

admin@ASW-7004:~$ sonic-cfggen -d --print-data | grep 'bgp_eoiu'
            "bgp_eoiu": "true",   <============ show the  config
admin@ASW-7004:~$ sonic-cfggen -d -v bgp_eoiu
                                               <============ show nothing

admin@ASW-7004:~$ sonic-cfggen -d -v 'WARM_RESTART' 
{'bgp': {'bgp_eoiu': 'true', 'bgp_timer': '120'}}
admin@ASW-7004:~$

Reply:
I just updated the diff. Please review it. thanks

Reply:
```
if [ $(sonic-cfggen -d -v 'WARM_RESTART.bgp.bgp_eoiu' | grep -c 'true')
```
to
```
if [ $(sonic-cfggen -d -v 'WARM_RESTART.bgp.bgp_eoiu' = 'true' ];
```
or
```
if [[ $(sonic-cfggen -d -v 'WARM_RESTART.bgp.bgp_eoiu' == 'true' ]];
```